batch/batch/driver/instance_pool.py

```diff
@@ -51,18 +53,21 @@ def __init__(self, app, machine_name_prefix):
 
         # pending and active
         self.live_free_cores_mcpu = 0
+        self.live_total_cores_mcpu = 0
```

Comment:
the comment above is now unclear. let's move it to an end-of-line comment to line 55.

Reply:
Why is it unclear? They're both computed from pending and active instances.

Reply:
Ah, yes, I left this comment before I figured out what live_total_cores_mcpu was, my bad.